### ISSUE TITLE
Provide output feedback when building images

### ISSUE THREAD

**Issue by davesann** [palletops/lein-uberimage#11]
Building images can take some time. Progress output would be a good thing.

**Comment by hugoduncan** [comment]
Indeed.  Will have to check if `clj-http` provides a mechanism for this.

**Comment by davesann** [comment]
yes I thought that one could be more tricky. I also found that the build is fast once the base images are available. Perhaps just recommending a docker pull of your base image before building is enough.
